rally/README.md

```diff
@@ -0,0 +1,39 @@
+# APM Rally Track
+
+For general information about rally tracks and how to use them, 
+check out the [docs](https://esrally.readthedocs.io/en/stable/index.html).
+and the [rally](https://github.com/elastic/rally) and [rally-tracks](https://github.com/elastic/rally-tracks) github 
+repos.
+
+## Corpora
+Data used in this track are dumped from example Opbeans applications, instrumented with `nodejs`, `python` and 
+`ruby` agents. The dump was created in September 2018. 
+You can also build your own corpora and reference them in the track.
+There is a [python script](https://github.com/elastic/apm-server/blob/d7b9d5027dd6a296792aa5179c0eaff8374d62d8/rally/fetch_data.py).
```

Comment:
will this link break when this is merged / branch deleted?  probably safe to change the sha to `master`